src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final double MAX_RANDOM_VALUE = 10.;
+    private static final int AMOUNT_OF_FIGURES = 5;
+
+    public static Figure getRandomFigure() {
```

Comment:
Don't use static methods in your solution. Static methods are generally a bad practice. Instead, you should create an instance of the `FigureSupplier` class and call its methods.